framework/components/ecl-site-switchers/_site-switchers.scss

```diff
@@ -1,128 +1,99 @@
-/*
-Site switcher
-
-Helps the user choose between political and informational websites.
-
-Markup: html/site-switcher.html
-
-.site-switcher--header - Header modifier
-.site-switcher--footer - Footer modifier
-
-Style guide: siteswitcher
+/**
+* Site switcher component
+* @define site-switcher
 */
 
-.site-switcher {
+.ecl-site-switcher {
   @include ecl-hidden-print();
 
-  display: block;
   background-color: map-get($ecl-colors, 'grey-5');
+  display: block;
   font-size: map-get($ecl-font-size, 's');
 }
 
-.site-switcher__list {
+.ecl-site-switcher__link {
+  display: block;
+  font-weight: bold;
+  line-height: 1.3;
+  padding: 0.5em 1.667em;
+  text-decoration: none;
+  width: 100%;
+
+  &::after,
+  &--is-external::after {
+    content: none !important;
+  }
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+}
+
+.ecl-site-switcher__list {
   list-style: none;
   margin: 0;
   padding: 0;
 }
 
-.site-switcher__option {
+.ecl-site-switcher__option {
   display: inline-block;
   float: left;
```

Comment:
We could have use display flex instead of float, but it should be ok too here (just have to check when the component is integrated in a real page )